runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java

```diff
@@ -76,8 +76,7 @@ public static FlinkRunner fromOptions(PipelineOptions options) {
     }
 
     if (flinkOptions.getFilesToStage() == null) {
-      flinkOptions.setFilesToStage(
-          detectClassPathResourcesToStage(FlinkRunner.class.getClassLoader()));
+      flinkOptions.setFilesToStage(detectClassPathResourcesToStage(FlinkRunner.class));
```

Comment:
This would step at FlinkRunner classloader, no? FlinkRunner is not part of the Flink cluster and needs to be shipped.

Reply:
No, the test explicitly stops at classloader of the given class (which is included, see [here](https://github.com/apache/beam/pull/9451/files#diff-c7786e41ca10361c918552b033e7cbffR63)).